scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+
+[Service]
+User=haas_user
+Group=haas_user
+WorkingDirectory=/var/lib/haas/
+ExecStart=/usr/bin/haas serve_networks
```

Comment:
So everything after this line has sane defaults afaik, can you explain why you've explicitly filled in things like `KillMode` and `ExecReload`? `Type=simple` probably does make sense even though it is the default, just for clarity.